docs/konnector_declaration_v3.md

```diff
@@ -28,6 +28,7 @@ remarks are welcome.
 {
   "version": "1.0.0",
     "name": "Trainline",
+    "type": "node",
```

Comment:
It is not so bad if we don't want to have to read in the package.json to run the konnector. Or we can run the konnector with npm start and let the developper of the konnector handle that part.